code/web/src/modules/user/ProfileForm.js

```diff
@@ -49,9 +49,13 @@ class ProfileForm extends Component {
       description = this.props.user.details.description
     }
 
+    if (this.props.user.details.image) {
```

Comment:
Not asking for a change here, but I'm curious if there is ever a situation in which an image will not be anything since we are setting the default state above.

Reply:
Yes, the `if` statement on line 52 checks if there is a user image in props and if so reassigns `image` to that value.